### ISSUE TITLE
Consider allowing a buyer review on an unfulfilled (but accepted) order

### ISSUE THREAD

**Issue by placer14** [OpenBazaar/openbazaar-desktop#1815]
There are cases where a physical good order which was paid without moderation to be accepted (causing the funds to transfer in an offline case) but is not fulfilled by the vendor. In these cases the buyer would like to leave a review for the vendor but is currently not permitted. Should we consider this use case and allow it?

**Comment by pinheadmz** [comment]
Agreed. But, are reviews stored on the vendor's node? Seems like a clever scammer could prevent negative reviews from being seen.

**Comment by placer14** [comment]
That's true, and while that issue is coupled with this one by design for now, I'd rather consider the concern on its own and then reconsider the design around it if needed... hopefully motivating a more orthogonal approach next time around. ;)
